mergify_engine/tasks/engine/actions_runner.py

```diff
@@ -31,13 +31,17 @@
 
 
 def get_already_merged_summary(ctxt, match):
-    for source in ctxt.sources:
-        if (
-            source["event_type"] != "pull_request"
-            or source["data"]["action"] != "closed"
-            or not ctxt.pull["merged"]
-        ):
-            return ""
+    if not (
+        ctxt.pull["merged"]
+        and any(
+            filter(
+                lambda s: s["event_type"] == "pull_request"
+                and s["data"]["action"] == "closed",
+                ctxt.sources,
+            )
```

Comment:
```suggestion
            s for s in ctxt.sources if s["event_type"] == "pull_request" and s["data"]["action'] == "closed"
```
Not a big deal but it's usually easier to read.

Reply:
done